index.src.html

```diff
@@ -519,33 +554,34 @@ <h3 id="integration-websockets">Integration with WebSockets</h3>
 
   <h3 id="integration-html">Integration with HTML</h3>
 
-  To support the checks in [[FETCH]], we store an <dfn>address space</dfn>
-  on both {{Document}} and {{WorkerGlobalScope}} objects, which is set as
-  follows during {{Document}} and {{Worker}} initialization:
+  To support the checks in [[FETCH]], we store an
+  <dfn>address space property</dfn> on both {{Document}} and
+  {{WorkerGlobalScope}} objects, which is set as follows during {{Document}} and
+  {{Worker}} initialization:
   
-  1.  Set the <a>address space</a> to "`local`" if the the resource used
-      to instantiate the {{Document}} or {{Worker}} was was delivered
+  1.  Set the <a>address space property</a> to "`local`" if the the resource
+      used to instantiate the {{Document}} or {{Worker}} was was delivered
       from a <a>local address</a>, or from a URL whose <a for="url">scheme</a>
       is "`file`" (see [[#file-url]]).
 
-  2.  Set the <a>address space</a> to "`private`" if the the resource used
-      to instantiate the {{Document}} or {{Worker}} was was delivered
+  2.  Set the <a>address space property</a> to "`private`" if the the resource
+      used to instantiate the {{Document}} or {{Worker}} was was delivered
       from a <a>private address</a>.
 
-  3.  Set the <a>address space</a> to "`public`" if the the resource used
-      to instantiate the {{Document}} or {{Worker}} was was delivered
+  3.  Set the <a>address space property</a> to "`public`" if the the resource
+      used to instantiate the {{Document}} or {{Worker}} was was delivered
```

Comment:
This section is going to need to be a little more detailed to point to the algorithms you'll actually want to change, but that rewrite might be worth deferring until after inheritance/container/etc work is further along.

Reply:
I agree. I thought I would land this patch first, then start hacking on better integration and inheritance.